Snippets/Snippets_6/Azure/Persistence/AzureStorage/Usage.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace Snippets6.Azure.Persistence.AzureStorage
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Persistence;
+    using NServiceBus.SagaPersisters;
+
+    class Usage
+    {
+        public void Demo()
+        {
+            #region PersistanceWithAzure 7
+
+            EndpointConfiguration endpointConfiguration = new EndpointConfiguration();
+            endpointConfiguration.UsePersistence<AzureStoragePersistence>();
+
+            #endregion
+        }
+
+        #region PersistenceWithAzureHost 7
+
+        public class EndpointConfig : IConfigureThisEndpoint
+        {
+            public void Customize(EndpointConfiguration configuration)
```

Comment:
can we name the param endpointConfiguration  to keep it consistent with all the other doco